### ISSUE TITLE
Browser Crash

### ISSUE THREAD

**Issue by BennyH26** [AnyChart/AnyChart#27]
The following HTML code generates a browser crash. Any ideas?

```
<!doctype html>
<head>
    <script src="https://cdn.anychart.com/releases/8.1.0/js/anychart-bundle.min.js"></script>
    <link rel="stylesheet" href="https://cdn.anychart.com/releases/8.1.0/css/anychart-ui.min.css"/>
    <link rel="stylesheet" href="https://cdn.anychart.com/releases/8.1.0/fonts/css/anychart-font.min.css"/>

    <style>
        html, body, #container {
            width: 100%;
            height: 100%;
            margin: 0;
            padding: 0;
        }
    </style>
</head>

<body>
<div id="container"></div>
</body>

<script>
    window.charting = {
        chart: anychart.stock(),
        dataTable: anychart.data.table()
    };

    window.charting.mapping = window.charting.dataTable.mapAs({
        'value': 1
    });

    window.charting.plot0 = window.charting.chart.plot(0);

    //Create series
    window.charting.series = window.charting.plot0.line(window.charting.mapping);

    //Set container and draw chart
    window.charting.chart.container('container');
    window.charting.chart.draw();

    //Set chart title
    window.charting.chart.title('Test Chart');

    window.charting.dataTable.addData([[1515698475780, 0.08826999]]);
    window.charting.dataTable.addData([[1515698479873, 0.08826998]]);
</script>

```

**Comment by loki20beta** [comment]
@BennyH26 we confirm. It looks like some kind of a bug that has something to do with the values passed. We will investigate and get back to you as soon as possible.

For reporting purposes:
OK: https://playground.anychart.com/CrgZlFDf/1
NOT OK: https://playground.anychart.com/CrgZlFDf/2

**Comment by BennyH26** [comment]
Much appreciated.

**Comment by maresha** [comment]
@BennyH26 this case was transferred to the dev team, please be assured I'll update you as soon as the bug will be fixed.

**Comment by jackvial** [comment]
+1

**Comment by Shestac92** [comment]
@BennyH26 @jackvial 
we glad to notify you that the fix for your issue is available with the 8.3.0 version update. Please, update your AnyChart binaries to get the fix. You can download all required binaries using our CDN - https://www.anychart.com/download/cdn/?v=8.3.0
